deeppavlov/models/kbqa/rel_ranking_bert_infer.py

```diff
@@ -1,190 +0,0 @@
-# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from logging import getLogger
-from typing import Tuple, List, Any, Optional
-
-from deeppavlov.core.common.registry import register
-from deeppavlov.core.models.component import Component
-from deeppavlov.core.models.serializable import Serializable
-from deeppavlov.core.common.file import load_pickle
-from deeppavlov.models.ranking.rel_ranker import RelRanker
-from deeppavlov.models.kbqa.wiki_parser import WikiParser
-from deeppavlov.models.preprocessors.bert_preprocessor import BertPreprocessor
-from deeppavlov.models.kbqa.sentence_answer import sentence_answer
-
-log = getLogger(__name__)
-
-
-@register('rel_ranking_bert_infer')
```

Comment:
mentioned in the deeppavlov/models/kbqa/sentence_answer.py line 31 and in the docs/features/models/kbqa.rst line 263